src/Symfony/Bundle/FrameworkBundle/Resources/config/messenger.xml

```diff
@@ -25,6 +25,12 @@
             <tag name="message_bus_middleware" priority="-10" />
         </service>
 
+        <service id="messenger.middleware.validator" class="Symfony\Component\Messenger\Middleware\ValidatingMiddleware">
+            <argument type="service" id="validator" />
+
+            <tag name="message_bus_middleware" priority="8" />
```

Comment:
Any reasoning for such priority? I'd argue it should be higher by default as it would be one of the most important "first steps".